beetsplug/edit.py

```diff
@@ -243,9 +244,9 @@ def edit_objects(self, objs, fields):
         old_data = [flatten(o, fields) for o in objs]
 
         # Set up a temporary file with the initial data for editing.
-        new = NamedTemporaryFile(mode='w', suffix='.yaml', delete=False)
+        new = NamedTemporaryFile(suffix='.yaml', delete=False)
         old_str = dump(old_data)
-        new.write(old_str)
+        new.write(old_str.encode('utf-8'))
```

Comment:
specifically 
`UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 7: ordinal not in range(128)`

I guess I'm a little confused as to what's expected here! If you can give me a hand on this one, then we're down to less than 10 total failures!